### ISSUE TITLE
Fails to import in python3

### ISSUE THREAD

**Issue by ManushB** [peakwinter/python-nginx#26]
**Version**: 1.4

**Exception**:
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/nginx/__init__.py", line 1, in <module>
    from main import *
ImportError: No module named 'main'
```
**Possible Fix**:
`from .main import *`

**Comment by fulder** [comment]
I think there has been something werid with the building of 1.4 version? The code looks different than master and even has files not existing at the tagged 1.4 version.

**Comment by peakwinter** [comment]
PyPI uploading problem -- this has been fixed in 1.4.1. Sorry!
